### DISCUSSION TITLE
Every time I want to update Copilot, I have to update Rider as well

### DISCUSSION THREAD

**Discussion by theReallyRealDusty** [community/community#47056] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

This is a problem because Rider updates seem to be more high risk than Copilot ones and I'd rather not update Rider if there's a chance something can be broken after.

**Comment by iamcjwilliams** [comment]
@lorinatzberger Visit [here](https://docs.github.com/en/copilot/troubleshooting-github-copilot) to troubleshoot your issue

**Reply by theReallyRealDusty** [reply]
I don't understand how that could be helpful. I posted some feedback about copilot updates always requiring the latest version of Rider and how that's a bad idea. Your link basically tells me how to gather logs and maybe look for some errors. These are not errors.

**Comment by iamcjwilliams** [comment]
@lorinatzberger Thank you, Our copilot team has been informed of your feedback
